### ISSUE TITLE
Not working 100% 

### ISSUE THREAD

**Issue by Latravant** [mikeyaworski/Auto-Claim-Twitch-Channel-Points#4]
It is collecting points for me but for the last few weeks it only seems to collect if I have twitch as current opened window. I used to listen to twitch in the background while playing mmorpgs, and it worked fine. But not now.
I've updated chrome. 
I’ve opened/closed the window. 
I only have one tab open, the twitch channel I want to collect points from.

**Comment by mikeyaworski** [comment]
I've tried to reproduce this and I can't on my setup. How do you know it's not collecting points? Is it just because your points aren't increasing, or because you see the button to claim points when you open the page again (so it was never clicked)? And when you play the video in the background, do you have audio for the tab playing or muted?

If it's the case that your points are not increasing as they should, but there's also no button to claim points for, then this extension wouldn't be able to do anything about it. It would be something along the lines of either

- A Chrome optimization to not play background video
- Twitch not awarding points for the background listening that you're doing

If my assumptions are correct, then some of the workarounds I can think of would be:

- Seeing if there are some Chrome settings to disable background player optimizations (unsure about this)
- Making sure the video playback is counting for Twitch by having the stream unmuted, but muting the tab (there is a way to do this FYI)
- Trying this on Firefox to test if it's Chrome causing your issue https://addons.mozilla.org/en-US/firefox/addon/auto-claim-twitch-chnnl-points

**Comment by Latravant** [comment]
I check my points daily and for 3 days it was stuck at the same number.
I have the twitch audio muted, but that never caused a problem before.
Last night it seemed to work somehow. The only thing I did is what I do
every day, reloaded the page. But now that I think about it, I also closed
the window and reopened it. Maybe it’s the google chrome updates causing
this? It updated 4 nights ago, then I clicked the relauch button. And
chrome seems to be getting ups every few weeks. The next time this happens
I’ll try and remember to see if it happens after an update or relaunch,
then close and reopen the window.

Thank you for the prompt response.
